src/middle/libraryjs.sk

```diff
@@ -13,6 +13,10 @@ const __imul fn(int, int) int = dynamic.Math.imul ? dynamic.Math.imul : (a, b) =
   return ((a as dynamic) * (b >>> 16) << 16) + (a as dynamic) * (b & 65535) | 0
 }
 
+const __createMap fn() dynamic = () => {
```

Comment:
I originally wrote this PR without this separate `__createMap` function. Maps were constructed inside `IntMap.new`, like so...

```
namespace IntMap {
  def new IntMap<T> {
    return dynamic.Map.new
  }
}
``` 

But this caused test failures in the minified build because code like `IntMap<int>.new.size` would compile to `new Map.size` which is not valid Javascript. I guessed that wrapping the Map construction in its own function would fix the issue and I I was right but I'm still nervous that there is a general issue where Skew code like `dynamic.Thing.new` isn't always wrapped in parens when it should be.